data/schema.graphql

```diff
@@ -10475,20 +10386,6 @@ type SearchAggregationResults {
   slice: SearchAggregation
 }
 
-# Search criteria
-type SearchCriteria {
-  artistID: String
-  attribution: String
-  category: String
-  internalID: ID!
-  priceMax: Int
-  priceMin: Int
-  size: String
-}
-
-# A search criteria or errors object
-union SearchCriteriaOrErrorsUnion = Errors | SearchCriteria
-
```

Comment:
Why are these schema updates in this PR? Rebase with master?

Reply:
fixed